spec/lib/catalog/determine_task_relevancy_spec.rb

```diff
@@ -62,7 +62,7 @@
       let(:status) { "ok" }
 
       context "when the task context has a key path of [:service_instance][:id]" do
-        let(:payload_context) { {"service_instance" => {"id" => "321"}} }
+        let(:payload_context) { {"service_instance" => {"id" => 321}} }
```

Comment:
I'm curious why this changed, it doesn't look like we're doing any sort of search for integers in the context and turning them into strings, the only change in the logic was for the `task_id` specifically, right?

Reply:
It does not directly impact what we are fixing now. From the actual message sent by topology this id is a number rather than a string.